web/app_options.js

```diff
@@ -168,6 +168,11 @@ const defaultOptions = {
     value: 2,
     kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
   },
+  capCanvasAreaFactor: {
+    /** @type {number} */
+    value: 200,
```

Comment:
Given that https://github.com/mozilla/pdf.js/pull/19755#issue-2967184930 now contains:
> The default value is 200% on desktop and will be 100% for GeckoView.

I'd expect that to be reflected in the definition here?

```suggestion
    value: (
      typeof PDFJSDev === "undefined"
        ? window.isGECKOVIEW
        : PDFJSDev.test("GECKOVIEW")
    )
      ? 100
      : 200,
```

Reply:
We've only one pref file in m-c, so we need to update `PdfJsOverridePrefs.js`, right ? or am I missing something ?

Reply:
Right, we don't have separate prefs-files for desktop/geckoview; I suppose that we can disregard the suggestion :-)